packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -62,12 +63,24 @@ const StoryPromoWrapper = styled.div`
       grid-column-gap: ${GEL_GUTTER_ABOVE_600PX};
     }
 
-    @media (min-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN}) {
-      grid-template-columns: repeat(12, 1fr);
-    }
+    ${({ topStory }) =>
+      topStory
+        ? css`
```

Comment:
I don't think the css tag function is required here. I remember Sareh running into problems when nesting tag functions. Template literals should do

Reply:
Ok, I will remove it.

Reply:
Done. I also removed another one that was far down.